### ISSUE TITLE
Ubuntu Host Installation

### ISSUE THREAD

**Issue by shainaraskin** [polyverse/katacoda-scenarios#2]
There is no explanation of how the host installation is different from the container installation. We may want to include a section that describes the difference.

**Comment by archisgore** [comment]
Agreed. The difference really is in speed of install (number of packages). Other than that there's not a great deal of difference. The host installation does demonstrate one key thing:
1. Reinstalling of system components (systemd, etc.) safely, because we utilize the package manager, who knows how to do it correctly.
